.travis.yml

```diff
@@ -24,28 +24,30 @@ env:
   - sonarqube: 8.2-community
 
 script:
+  - IS_JDK_11=$(java -version 2>&1 |grep "^openjdk version \"11\." > /dev/null ; echo $?)
```

Comment:
Did you try to use `TRAVIS_JDK_VERSION` (https://docs.travis-ci.com/user/environment-variables) ?

Reply:
Actually, I missed this variable while reading the Travis documentation.
I will give it a try as it's way simpler...

Reply:
It works, as you'll see in the next commit. I resolve this conversation.